Library/Formula/bigloo.rb

```diff
@@ -45,4 +48,8 @@ def install
     manpages = %w[bgldepend bglmake bglpp bgltags bglafile bgljfile bglmco bglprof]
     manpages.each { |m| man1.install "manuals/#{m}.man" => "#{m}.1" }
   end
+
+  test do
+    assert_match "Bigloo (4", shell_output("bigloo --help")
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!